README.rst

```diff
@@ -14,7 +14,7 @@
 
 |
 
-Ray is a unified framework for scaling AI and Python applications. Ray consists of a core distributed runtime and a toolkit of libraries (Ray AIR) for accelerating ML workloads:
+Ray is a unified framework for scaling AI and Python applications. Ray consists of a core distributed runtime and a toolkit of libraries (Ray AIR) for simplifying ML compute:
```

Comment:
a bit worried that "ML compute" isn't a common term for people; 

can you do ML compute workloads?

Reply:
I don't think so, that's much less memorable and a bit of a mouthful. ML Compute is clear and unambiguous.